vnext/Shared/OInstance.cpp

```diff
@@ -579,6 +580,10 @@ void InstanceImpl::loadBundleInternal(std::string &&jsBundleRelativePath, bool s
       m_innerInstance->loadScriptFromString(std::move(bundleString), jsBundleRelativePath, synchronously);
 #endif
 
+#if defined(_CHAKRACORE_H_)
+    } catch (const facebook::react::ChakraJSException &e) {
+      m_devSettings->errorCallback(std::string{e.what()} + "\r\n" + e.getStack());
```

Comment:
Do we publicly expose this error string anywhere currently?

Reply:
What do you mean by "publicly"? This message is sent to the react host, and eventually to the ReactOption's owner. Each host app handles it differently.

Reply:
E.g. do we have something in Office or elsewhere that has a callback that will display that error string publicly? We ideally shouldn't, but if we do, this could leak more information than was originally intended when they wrote the error callback.

Reply:
We shouldn't be directly exposing these error messages to users anyway.  The callback is primarily a developer experience thing. -- The error messages already coming though here are not end user friendly.

Reply:
Yup, and furthermore this error message is not localized. So it shouldn't be shown to users.